tests/test_num_params.py

```diff
@@ -76,6 +81,28 @@ def get_models(workload):
                                                jnp.float32))["params"]
     # Init PyTorch model.
     pytorch_model = PyTorchResNet_c10(num_classes=10)
+
+  elif workload == 'criteo1tb':
+    # Init Jax model.
+    input_shape = (1, 39)
+    model_init = JaxDlrmSmall(
+        vocab_sizes=tuple([1024 * 128] * 26),
+        total_vocab_sizes=sum(tuple([1024 * 128] * 26)),
+        num_dense_features=13,
+        mlp_bottom_dims=(128, 128),
+        mlp_top_dims=(256, 128, 1),
+        embed_dim=64).init
```

Comment:
can we make these kwargs a dict and pass them via **kwargs to both jax and pytorch versions?